injector/src/test/java/edu/ucr/cs/riple/injector/RemovalTest.java

```diff
@@ -162,7 +162,7 @@ public void removeMethodInlineAnnot() {
             "package edu.ucr;",
             "import edu.custom.Annot;",
             "public class Test {",
-            "   public@Annot Object test() {",
+            "   public @Annot Object test() {",
```

Comment:
Is this change because we changed how we insert annotations to have a leading space?

Reply:
Yes exactly. With new implementation we reduced scope of annotations Injector can remove since we only remove annotations the injector added itself. And the `@Annot` in this code is not how Injector adds an annotation.